src/modules/expense/Form.tsx

```diff
@@ -0,0 +1,524 @@
+          </FormControl>
+          <FormControl>
+            <InputLabel id='select-end-after'>End After</InputLabel>
+            <Select
+              labelId='select-end-after'
+              id='simple-select-end-after'
+              value={selectedEndAfter}
+              onChange={handleChangeEndAfter}
+              input={
+                <OutlinedInput id='select-single-end-after' label='end-after' />
+              }
+            >
+              <MenuItem value='Date'>Date</MenuItem>
+            </Select>
+          </FormControl>
+          <Button
+            onClick={handleFirstStepNextButton}
+            variant='contained'
+            sx={{
+              padding: theme.spacing(2),
+              textTransform: 'none',
+              fontSize: '16px',
+            }}
+          >
+            Next
+          </Button>
+        </StyledRepeatOptionsSection>
+      )}
+      {showRepeatDetails && (
+        <StyledRepeatDetailsSection>
+          <Stack>
+            <FormControl>
+              <InputLabel id='select-frequency'>Frequency</InputLabel>
+              <Select
+                labelId='select-frequency'
+                id='simple-select-frequency'
+                value={selectedFrequency}
+                onChange={handleChangeFrequency}
+                input={
+                  <OutlinedInput
+                    id='select-single-frequency'
+                    label='frequency'
+                  />
+                }
+              >
+                <MenuItem value='Yearly'>Yearly</MenuItem>
+                <MenuItem value='Weekly'>Weekly</MenuItem>
+                <MenuItem value='Monthly'>Monthly</MenuItem>
+                <MenuItem value='Yearly'>Yearly</MenuItem>
+              </Select>
+            </FormControl>
+          </Stack>
+          <Stack
+            sx={{
+              display: 'flex',
+              flexDirection: 'row',
+              justifyContent: 'space-between',
+              gap: theme.spacing(2),
+            }}
+          >
+            <FormControl sx={{ width: '50%' }}>
+              <InputLabel id='select-end-after'>End After</InputLabel>
+              <Select
+                labelId='select-end-after'
+                id='simple-select-end-after'
+                value={selectedEndAfter}
+                onChange={handleChangeEndAfter}
+                input={
+                  <OutlinedInput
+                    id='select-single-end-after'
+                    label='end-after'
+                  />
+                }
+              >
+                <MenuItem value='Date'>Date</MenuItem>
+              </Select>
+            </FormControl>
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DatePicker
+                sx={{ width: '50%' }}
+                value={date}
+                onChange={(newDate) => setDate(newDate)}
+              />
+            </LocalizationProvider>
+          </Stack>
+          <Button
```

Comment:
Same here about using button from shared components

Reply:
There's a Button component in the shared-components folder, use it instead of using Button component directly from MUI :)